posts/2020-10-23-beta-20.0.0.12.adoc

```diff
@@ -0,0 +1,440 @@
+    @Complete
+    @Path("/complete")
+    @PUT
+    public Response completeAction(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId)
+    {
+        /*
+         * Business logic to cleanup resources once the LRA has closed successfully
+         */
+         return Response.ok().build();
+    }  
+----
+
+Should a service in an LRA be canceled or timeout, the `@Compensate` methods for each participating service in the LRA will be called:
+
+[source, java]
+----
+    @Compensate
+    @Path("/compensate")
+    @PUT
+    public Response compensateAction(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId)
+    {
+        /*
+         * As the LRA has been canceled business logic to compensate any changes is necessary
+         */
+         return Response.ok().build();
+    } 
+----
+
+To enable the LRA Coordinator feature in your app, add the following to your `server.xml`:
+[source, xml]
+----
+<featureManager>
+    <feature>cdi-2.0</feature>
+    <feature>jaxrs-2.1</feature>
+    <feature>mpLRACoordinator-1.0</feature>
+</featureManager>
+----
+
+To enable the LRA Participants feature in your app, add the following to your `server.xml`:
+[source, xml]
+----
+<featureManager>
+    <feature>cdi-2.0</feature>
+    <feature>jaxrs-2.1</feature>
+    <feature>mpLRA-1.0</feature>
+</featureManager>
+----
+
+For more information:
+
+link:https://github.com/eclipse/microprofile-lra/blob/master/spec/src/main/asciidoc/microprofile-lra-spec.adoc[Microprofile Long Running Actions Specification]
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>20.0.0.12-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[20.0.0.12-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in Jakarta EE is the names of packages changing to accommodate the new `jakarta.*` namespace. In this Open Liberty beta, we have a number of new API Release Candidates to join the expanding library of supported Jakarta packages.
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* Enterprise Beans Remote Client 2.0 (`ejbRemoteClient-2.0`)
+* RESTful Web Services 3.0 (`restfulWS-3.0` and `restfulWSClient-3.0`)
+* Jakarta Server Faces 3.0 (`faces-3.0`)
+* Connectors 2.0 (`connectors-2.0`)
```

Comment:
Can be changed to:
```
Jakarta Connectors 2.0 (`connectors-2.0`)
```
To be consistent with the other feature's in this announcement.

Reply:
Added, thanks.